addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -100,6 +100,7 @@
 						</control>
 						<include content="MediaFlag">
 							<param name="texture" value="flags/flag.png" />
+							<param name="visible" value="!String.IsEmpty(MusicPlayer.SampleRate)" />
 						</include>
```

Comment:
you'll need to apply a visible condition to the parent group instead,
as the group will still be taking up space when only hiding the flag.